powerscore-entryboard/src/components/editboard/Editboard.vue

```diff
@@ -68,12 +47,17 @@ function toggleEditMode() {
   state.editMode = !state.editMode;
 }
 
+onMounted( () => {
```

Comment:
What was the verdict on marking this callback as `async` and calling `await` on `refreshTeams()`?